sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/Context.scala

```diff
@@ -16,5 +16,5 @@
 
 package com.akkaserverless.scalasdk
 
-//FIXME
+//FIXME (not extend from javasdk Context?)
```

Comment:
yes, I guess that shouldn't be needed since the adapters create new instances, but can be cleanup in another PR